doc/developer_guide/conf.py

```diff
@@ -275,8 +277,10 @@
 
 # Bibliographic Dublin Core info.
 epub_title = u"PSyclone Developer Guide"
-epub_author = u'Rupert Ford, Joerg Henrichs, Iva Kavcic, Andrew Porter and Sergi Siso'
-epub_publisher = u'Rupert Ford, Joerg Henrichs, Iva Kavcic, Andrew Porter and Sergi Siso'
+epub_author = u'Rupert Ford, Joerg Henrichs, Iva Kavcic, Andrew Porter and ' \
+    'Sergi Siso'
+epub_publisher = u'Rupert Ford, Joerg Henrichs, Iva Kavcic, Andrew Porter ' \
+    'and Sergi Siso'
```

Comment:
Please could you update the copyright dates throughout this file.

Reply:
Done